cloud/endagaweb/models.py

```diff
@@ -1432,14 +1469,18 @@ def create_auth(sender, instance, created, **kwargs):
         authenticate.
         """
         if not instance.auth_group or not instance.auth_user:
-            instance.auth_group, created_group = Group.objects.get_or_create(name='network_%s'
-                % instance.pk)
+            # instance.auth_group, created_group = Group.objects.get_or_create(name='network_%s'
+            #     % instance.pk)
+            instance.auth_group, created_group = Group.objects.get_or_create(name='%s_GROUP_%s'
+                % (instance.name, instance.pk))
```

Comment:
convention would be to put the `%` at the end of the previous line (and make sure indentation of this line is correct).